### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by jakobbraun**
Dependency updates

### CODE REVIEW

File: doc/changes/changes_2.0.1.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Virtual Schema for Redshift 2.0.1, released 2021-11-23
+
+Code name: Dependency Updates on 'Remove `SQL_DIALECT` property'
```
Comment (AnastasiiaSergienko) [inline doc/changes/changes_2.0.1.md]: Why "on 'Remove `SQL_DIALECT` property'" ? Maybe rephrase the code name

File: src/main/java/com/exasol/adapter/dialects/redshift/RedshiftSqlDialect.java [hunk 2]
```diff
@@ -123,7 +124,8 @@ public String getStringLiteral(final String value) {
             return "NULL";
         } else {
             if (value.contains("'") || value.contains("\\")) {
-                throw new IllegalArgumentException("Redshift string literal contains illegal characters: ' or \\.");
+                throw new IllegalArgumentException(ExaError.messageBuilder("E-VS-RS-1")
```
Comment (AnastasiiaSergienko) [inline src/main/java/com/exasol/adapter/dialects/redshift/RedshiftSqlDialect.java]: I would use REDSH or RDSH instead of RS, because it gives more hint which VS it is

File: doc/changes/changes_2.0.1.md (hunk 1 above)
Comment (jakobbraun) [reply doc/changes/changes_2.0.1.md]: that was the name of the last release.
@redcatbear suggested doing `Dependency Updates on <Last release name>` a while ago.

**Review by AnastasiiaSergienko — APPROVED** [review]
